components/layout_2020/flow/line.rs

```diff
@@ -302,11 +302,6 @@ impl InlineBoxLineItem {
             border.inline_start = Length::zero();
             margin.inline_start = Length::zero();
         }
-        if !self.is_last_fragment_of_ib_split {
-            padding.inline_end = Length::zero();
-            border.inline_end = Length::zero();
-            margin.inline_end = Length::zero();
-        }
```

Comment:
I'm not sure why this can be removed, but nothing is failing, so...

Reply:
I neglected to mention this in the commit message, but it's a related cleanup. The borders are also zeroed out at the bottom of the same function so these lines weren't really doing anything.